app/budget/services/actions/budgetActions.js

```diff
@@ -39,6 +39,39 @@ class BudgetActions {
 						}
 					}
 
+					var sectionGroupCosts = results.sectionGroupCosts;
+					var terms = ["05", "07", "10", "01", "03"];
+
+					var subjectCode = results.courses[0].subjectCode;
+					var termCodes = terms.map(function(term) {
+						if (term == "01" || term == "03") {
```

Comment:
Two issues:
- Logic for translating an academic year and term into a termCode is something we do a fair amount of, I suspect a helper already exists to perform this logic (or if it doesn't, this logic should live in a helper).
- This logic is tricky (which is why I like to not write it by hand each time :P), and I don't think you have it quite right. you can see all 10 term codes here:
https://registrar.ucdavis.edu/faculty-staff/scheduling-guide/codes

I believe terms 01, 02, and 03 should have their years incremented.